test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -373,6 +384,41 @@ public void createParticipantEmailEnabledVerificationWanted() {
         
         verify(accountWorkflowService).sendEmailVerificationToken(any(), any(), any());
         verify(account).setStatus(AccountStatus.UNVERIFIED);
+        verify(account).setEmailVerified(Boolean.FALSE);
+    }
+    
+    @Test
+    public void createParticipantWithPhoneEmailEnabledVerificationWanted() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval(null, null);
+
+        StudyParticipant phoneOnly = new StudyParticipant.Builder()
+                .copyOf(PARTICIPANT).withEmail(null).build();
+        
+        participantService.createParticipant(STUDY, CALLER_ROLES, phoneOnly, true);
+        
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
+        // The account will not be verified.
+        verify(account).setStatus(AccountStatus.UNVERIFIED);
+        verify(account).setPhone(PHONE);
+        verify(account).setPhoneVerified(Boolean.FALSE);
+    }
+    
+    @Test
+    public void createParticipantWithPhoneEmailEnabledVerificationNotWanted() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval(null, null);
+
+        StudyParticipant phoneOnly = new StudyParticipant.Builder()
+                .copyOf(PARTICIPANT).withEmail(null).build();
+        
+        participantService.createParticipant(STUDY, CALLER_ROLES, phoneOnly, false);
+        
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
+        // The account will not be verified.
+        verify(account).setStatus(AccountStatus.ENABLED);
+        verify(account).setPhone(PHONE);
+        verify(account).setPhoneVerified(Boolean.FALSE);
```

Comment:
This is actually not covered by createParticipantEmailEnabledVerificationWanted(). That's an entirely separate if-branch.

As far as I can tell, this is the only test that covers the branch where we don't ask for email verification and we don't submit an email address. We have an if-check that makes sure we don't call setEmailVerified(true), and as far as I can tell, there's nothing in this unit test that verifies that that's (not) happening.